actionRuntimes/ruby2.5Action/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM openwhisk/action-ruby-v2.5
```

Comment:
With #3680 we won’t need this (and can remove all the others like it). 

cc @csantanapr

Reply:
Ok. I will see for how #3680 goes then.